README.md

```diff
@@ -6,7 +6,7 @@ Installation
 
 Until this repo is published on NPM
 Install via
-`npm install --save git+ssh://git@github.com/dronedeploy/dronedeploy-functions-api.git`
+`npm install --save git+ssh://git@github.com/dronedeploy/function-wrapper.git`
```

Comment:
Technically, this instruction is out of date now.  It would be `npm install --save @dronedeploy/function-wrapper`